content/applications/general/calendars/outlook/outlook_calendar.rst

```diff
@@ -1,103 +1,149 @@
-   | For the **Redirect URI restrictions**, copy your Odoo database URI followed by
-     ``/microsoft_account/authentication``. Example:
+   For the :guilabel:`Redirect URI restrictions`, copy the Odoo database URI (URL) followed by
+   `/microsoft_account/authentication`. For example, 
+   `(https://www.companyname.odoo.com/microsoft_account/authentication)`.
+
+   .. image:: outlook_calendar/redirect_URIs.png
+      :align: center
+      :alt: Odoo's database URI that is accepted when Microsoft returns authentication.
 
-      .. image:: outlook_calendar/redirect_URIs.png
-         :align: center
-         :alt: Odoo's database URI that is accepted when microsoft returns authentication
+   For more information on the restrictions and limitations of URIs, `check this page <https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
 
-   | For more information on the restrictions and limitations of URIs, `check this page <https://docs.microsoft.com/en-us/azure/active-directory/develop/reply-url>`_.
+Regarding credentials, the user **must** :guilabel:`Add a client secret`, which allows Odoo to
+authenticate itself, requiring no interaction from the user's side. :guilabel:`Certificates` are
+optional.
 
-.. note::
-   Regarding credentials, you *must* **Add a client secret**, called *Client Secret* in Odoo, which
-   allows Odoo to authenticate itself, requiring no interaction from your side. *Certificates* are
-   optional.
+Since resetting the synchronization can be tricky, Odoo recommends setting the maximum allowed
+:guilabel:`expiration date` for the Client Secret (24 months), so there is no need to
+re-synchronize soon. 
 
 Configuration in Odoo
 =====================
 
-Go to general *Settings* and activate the *Outlook Calendar* feature.
+In the Odoo database, go to :menuselection:`Settings --> General Settings` and activate the
+:guilabel:`Outlook Calendar` feature.
 
 .. image:: outlook_calendar/outlook_calendar.png
    :align: center
-   :alt: Outlook Calendar feature activated in Odoo
+   :alt: Outlook Calendar feature activated in Odoo.
 
-From your Azure portal, under *Overview*, copy your *Application (client) ID*, and paste it to
-*Client ID* in Odoo.
+From the Microsoft Azure portal, under the :guilabel:`Overview` section, copy the
+:guilabel:`Application (Client) ID`, and paste it into the :guilabel:`Client ID` field in Odoo.
 
 .. image:: outlook_calendar/application_client_ID.png
    :align: center
-   :alt: Client ID token to be copied from Microsoft to Odoo
+   :alt: Client ID token to be copied from Microsoft to Odoo.
 
-Going back to your Azure portal, now under *Certificates & secrets*, copy your *Client Secret Value*
-and paste it to *Client Secret* in Odoo.
+In the Microsoft Azure portal, under the :guilabel:`Certificates & secrets` section, copy the
+:guilabel:`Client Secret Value` and paste it into the :guilabel:`Client Secret` field in Odoo.
 
 .. image:: outlook_calendar/client_secrets.png
```

Comment:
[hypens](https://www.odoo.com/documentation/15.0/contributing/documentation/content_guidelines.html#media-files) > underscores in media file name.